scss/_variables.scss

```diff
@@ -76,4 +76,4 @@ $o-buttons-themes: (
 /// Icon names as used in fticons
 ///
 /// @type List
-$o-buttons-icons: "arrow-left", "arrow-right" !default;
```

Comment:
any reason why the default set has changed? I agree a few in addition to left and right would be good, but would add cross and remove upload from this list

Reply:
Lily has asked for upload as she needs it for a project, and the other two are myFT